### PR TITLE
[hplsql] syntax highlight for hplsql dialect

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- syntax highlight for hplsql keywords, built-in functions, datatypes.

## How was this patch tested?

- manually check with editor.

### CODE REVIEW

File: desktop/libs/notebook/src/notebook/templates/editor_components.mako [hunk 1]
```diff
@@ -2088,6 +2088,12 @@ ${ sqlSyntaxDropdown.sqlSyntaxDropdown() }
         snippetImage: '${ static("beeswax/art/icon_beeswax_48.png") }',
         sqlDialect: true
       },
+      hplsql: {
+        placeHolder: '${ _("Example: SELECT * FROM tablename, or press CTRL + space") }',
```
Comment (JohanAhlen) [inline desktop/libs/notebook/src/notebook/templates/editor_components.mako:2092]: How about an HPL/SQL example here? For instance: `placeHolder: 'Example: PRINT \'Hello world\''`

**Comment by agl29** [comment]
some places, we are using _snippetviewsetting_ :
1 -> https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L2960
2 -> https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/js/ko/components/ko.historyPanel.js#L179
3 -> https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/templates/editor2.mako#L1057
4 -> https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/templates/editor_m.mako#L180

so all files also need to update with 'hplsql' settings? @JohanAhlen

**Comment by JohanAhlen** [comment]
> https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L2960



> some places, we are using _snippetviewsetting_ : 1 -> https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L2960 2 -> https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/js/ko/components/ko.historyPanel.js#L179 3 -> https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/templates/editor2.mako#L1057 4 -> https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/templates/editor_m.mako#L180
> 
> so all files also need to update with 'hplsql' settings? @JohanAhlen

No, I think it's enough with editor_components.mako, the other ones are probably just unused duplication.

**Review by JohanAhlen — APPROVED** [review]
Nice!

**Pull request merged**